oak/server/channel.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OAK_SERVER_CHANNEL_H_
+#define OAK_SERVER_CHANNEL_H_
+
+namespace oak {
+
+// Abstract interface for Oak communication channels.
```

Comment:
Should note that this is for a single method call.

Reply:
Done.